apps/web/src/stores/users.ts

```diff
@@ -0,0 +1,27 @@
+import { User } from '@latitude-data/core'
+import { useToast } from '@latitude-data/web-ui'
+import { getUsersActions } from '$/actions/users/fetch'
+import useSWR, { SWRConfiguration } from 'swr'
+
+export default function useUsers(opts?: SWRConfiguration) {
+  const { toast } = useToast()
+
+  const fetcher = async () => {
+    const [data, error] = await getUsersActions()
```

Comment:
I'll open a light debate here. For simple crud screens do we need to store data in the client? Couldn't we just `revaltidatePath` in the action?

Reply:
I don't think it's a good idea to introduce more than one way data flows between frontend and backend, let's keep it simple, data flows always via actions.